src/events/components/SearchModule/DateRangeInput.tsx

```diff
@@ -0,0 +1,109 @@
+import { DateTime } from 'luxon';
+import React, { FC, useRef } from 'react';
+import { DayPickerProps } from 'react-day-picker';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+import 'react-day-picker/lib/style.css';
+
+import style from './search.less';
+
+const WEEKDAYS_SHORT = ['Sø', 'Ma', 'Ti', 'On', 'To', 'Fr', 'Lø'];
+const MONTHS = [
+  'Januar',
+  'Februar',
+  'Mars',
+  'April',
+  'Mai',
+  'Juni',
+  'Juli',
+  'August',
+  'September',
+  'Oktober',
+  'November',
+  'Desember',
+];
+
+const WEEKDAYS_LONG = ['Søndag', 'Mandag', 'Tirsdag', 'Onsdag', 'Torsdag', 'Fredag', 'Lørdag'];
+
+const FIRST_DAY_OF_WEEK = 1;
+
+const LABELS = { nextMonth: 'Neste måned', previousMonth: 'Forrige måned' };
+
+export interface IProps {
+  dateStart: DateTime;
+  dateEnd: DateTime;
+  handleResetClick: () => void;
+  handleToDateClick: (day: DateTime) => void;
+  handleFromDateClick: (day: DateTime) => void;
+}
+
+export const DateRangeInput: FC<IProps> = ({
+  dateStart: dateTimeStart,
+  dateEnd: dateTimeEnd,
+  handleToDateClick,
+  handleFromDateClick,
+  handleResetClick,
+}) => {
+  const dateStart = dateTimeStart.toJSDate();
+  const dateEnd = dateTimeEnd.toJSDate();
+
+  const modifiers = { start: dateStart, end: dateEnd };
+  const inputRef = useRef<DayPickerInput>(null);
+
+  const fromMonthProps: DayPickerProps = {
+    selectedDays: [dateStart, { from: dateStart, to: dateEnd }],
+    disabledDays: { after: dateEnd },
+    toMonth: dateEnd,
+    modifiers,
+    numberOfMonths: 2,
+    onDayClick: () => {
+      if (inputRef.current) {
+        inputRef.current.getInput().focus();
+      }
+    },
+    locale: 'no',
+    months: MONTHS,
+    weekdaysLong: WEEKDAYS_LONG,
+    weekdaysShort: WEEKDAYS_SHORT,
+    firstDayOfWeek: FIRST_DAY_OF_WEEK,
+    labels: LABELS,
+  };
+
+  const toMonthProps: DayPickerProps = {
+    selectedDays: [dateStart, { from: dateStart, to: dateEnd }],
+    disabledDays: { before: dateStart },
+    modifiers,
+    month: dateStart,
+    fromMonth: dateStart,
+    numberOfMonths: 2,
+    locale: 'no',
+    months: MONTHS,
+    weekdaysLong: WEEKDAYS_LONG,
+    weekdaysShort: WEEKDAYS_SHORT,
+    firstDayOfWeek: FIRST_DAY_OF_WEEK,
+    labels: LABELS,
+  };
+
+  return (
+    <div className={style.inputFromTo}>
+      <button onClick={handleResetClick}>Reset</button>
```

Comment:
The reset button does not seem to work when I've tested it. However I'm not sure if we really need a reset button?

Reply:
I think  it would be nice to have, but not necessary, although I am more worried about why it isn't working, because even if I make it set the parameters to `null`, it only removes dateEnd, not dateStart